sycl/source/detail/program_manager/program_manager.hpp

```diff
@@ -224,6 +248,11 @@ class ProgramManager {
 
   /// True iff a SPIR-V file has been specified with an environment variable
   bool m_UseSpvFile = false;
+
+public:
+  std::unordered_map<KernelSetId,
+                     std::unique_ptr<std::vector<RTDeviceBinaryImageUPtr>>> &
+  GetDeviceImages();
```

Comment:
What's the reason for adding this function?

Reply:
Removed.